cuenca/resources/base.py

```diff
@@ -77,6 +79,32 @@ def _update(
         return cls._from_dict(resp)
 
 
+@dataclass
+class Downloadable(Resource):
+    @classmethod
+    def download(
+        cls,
+        instance,
+        *,
+        session: Session = global_session,
+        file_format: FileFormat,
+    ) -> BytesIO:
+        resp = session.download(
+            f'/{cls._resource}/{instance.id}',
+            headers=dict(Accept=file_format.value),
+            stream=True,
```

Comment:
Does this actually do anything? IIRC, `request.content` consumes the stream

Reply:
?

Reply:
Is that stream is used when you want to bring the file at the time of `response.content`, but as we do immediately after the response, it is not necessary.